src/rules/add-claim-based-on-relationships.js

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
Can you rename the file to include `scaled-access` in the name?

Reply:
Done